Jenkinsfile

```diff
@@ -189,11 +189,7 @@ stage('Publish') {
                     """
                     sh "scripts/create-github-release.js"
 
-                    // Publish the updated package to the npm registry.
```

Comment:
Based on this https://github.com/yarnpkg/yarn/pull/5108, I modified the command.

I tested it and I confirm it works as expected. 

I removed the outdated comment and I simplified the way NPM_TOKEN is passed to the yarn command.